.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": "${workspaceFolder}/docs/source"
+}
```

Comment:
Was the inclusion of this file intentional? My guess is `.vscode/` should be added to `.gitignore`.